projects/dates/src/lib/adapters/primary/ui/today-date.component.css

```diff
@@ -0,0 +1,23 @@
+.today-text {
```

Comment:
Please don't use own styling. All this can be done with utility bootstrap classes